mobile/library/java/org/chromium/net/impl/NativeCronvoyEngineBuilderImpl.java

```diff
@@ -227,6 +232,54 @@ public NativeCronvoyEngineBuilderImpl setUpstreamTlsSni(String sni) {
     return this;
   }
 
+  /**
+   * Set whether to use a platform specific APIs to create UDP socket and the associated QUIC packet
+   * writer. Note that `setUseV2NetworkMonitor()` also needs to be called to take effect. This is a
+   * temporary API which will be deprecated once the platform specific extension is verified to work
+   * and will be used as the default.
+   */
+  public NativeCronvoyEngineBuilderImpl setUseQuicPlatformPacketWriter(boolean use) {
+    mUseQuicPlatformPacketWriter = use;
+    return this;
+  }
+
+  /**
+   * Set whether to enable QUIC connection migration across different network interfaces.
+   * Note that `setUseV2NetworkMonitor()` also needs to be called to take effect.
+   * If enabled, the engine will automatically be configured to use platform packet writer. *
+   */
+  public NativeCronvoyEngineBuilderImpl setEnableConnectionMigration(boolean enable) {
```

Comment:
Since this and the following methods are QUIC-specific, should they have QUIC in the name?